storio-common-annotations-processor/src/test/java/com/pushtorefresh/storio/common/annotations/processor/introspection/AnnotatedFieldValidationTest.java

```diff
@@ -30,15 +30,15 @@ public void failIfEnclosingElementIsNotType() {
         stub.processor.validateAnnotatedFieldOrMethod(stub.field);
     }
 
-    @Test
-    public void failIfThereIsNoAnnotation() {
-        AnnotationProcessorStub stub = AnnotationProcessorStub.newInstance();
-        when(stub.enclosingElement.getAnnotation(AnnotationProcessorStub.TestClassAnnotation.class)).thenReturn(null);
-
-        expectedException.expect(ProcessingException.class);
-        expectedException.expectMessage("Please annotate class TestClass with TestClassAnnotation");
-        stub.processor.validateAnnotatedFieldOrMethod(stub.field);
-    }
+//    @Test
```

Comment:
I really like how annotation processor tests are organised in [Epoxy](https://github.com/google/compile-testing) using [Google Compile Testing](https://github.com/airbnb/epoxy). I tried to set up it in StorIO couple of months ago but failed. Maybe it requires more investigation.